test/eredis_test_utils.erl

```diff
@@ -13,36 +13,29 @@
 -export([ start_server/2 ]).
 
 %% Get available TCP ports (includes TLS)
--spec get_tcp_ports() -> [erlang:port()].
```

Comment:
Why delete the specs?

Reply:
Is it beneficial to maintain specs on test utility code?
Both start_server/1 and stop_server/1 have wrong specs, do you like them to be corrected instead?

Reply:
If they're wrong, better to delete them than to keep them.

Is this one wrong though? It returns `[Port || Port <- erlang:ports(), ...]`, which looks like it has type `[erlang:port()]`. The spec kinda documents that it returns erlang ports, rather than integer port numbers.

We can delete the other specs that are wrong, like for start_server I guess, but I don't see why we should delete this one. Correct specs might maybe even better, but it's more work, so lets focus on deleting the incorrect ones first.

(In theory, if the tests are complex, it can potentially be useful to have type specs there too and to check them, but we're not there.)

Reply:
Corrected the faulty specs instead.